lib/active_model_serializers/serializable_resource.rb

```diff
@@ -49,7 +49,7 @@ def serializer_instance
 
     # Get serializer either explicitly :serializer or implicitly from resource
     # Remove :serializer key from serializer_opts
-    # Replace :serializer key with :each_serializer if present
+    # Replace :each_serializer key with :serializer if present
```

Comment:
Hmm, I don't think it's exactly clear either way, now that I look at it.  Should be

> Remove `:each_serializer` if present and set as `:serializer` key